nav2_amcl/src/amcl_node.cpp

```diff
@@ -2001,6 +2001,57 @@ void AmclNode::healthCheck(){
   }
 }
 
+void AmclNode::integrateOdometricChange(
```

Comment:
I might be annoying but reading `integrateOdometryChange` makes me think about calculating velocity. This method "shifts" in time a pose instead, right?

Reply:
Yes, sort of "shifts"

If you have a non-confusing name for that, I'm up to it.

I've chosen that name because of [this](https://github.com/cmrobotics/navigation2/blob/9773d31b26e2466a9dd12ff0d6c92804aef52783/nav2_amcl/src/amcl_node.cpp#L732) comment.

Reply:
Ah, integrating in the sense of adding and not a numeric integration of a physical entity. Fine then.